### PR TITLE
git: new article

### PR DESCRIPTION
**Pull request by mcpcpc**

### CODE REVIEW

File: vcs/git.txt [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+$ git checkout master
+$ git merge upstream/master
+
+
+References
+________________________________________________________________________________
+
+[0] https://git-scm.com
+
```
Comment (dylanaraps) [inline vcs/git.txt:100]: Drop this newline.

**Comment by dylanaraps** [comment]
This is probably better under `software` as well. The software `index.txt` file can be broken down into "cosmetic" subsections later on.

File: vcs/git.txt [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+Git is a free and open source distributed version control system designed to 
+handle everything from small to very large projects with speed and efficiency.
+
+
+Commands
+________________________________________________________________________________
+
+New KISS users may have various levels of exposure to git and other forms of
+version control systems. The following is intended to be "quick" reference sheet
```
Comment (dylanaraps) [inline vcs/git.txt:12]: `to be` -> `to be a`

File: vcs/git.txt [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+| Rebase           | $ git rebase                                              |
+|                  | # reapply commits on top of another base tip              |
+|                  |                                                           |
++------------------------------------------------------------------------------+
+
+
+Example 1: Staying Up-to-Date [1]
+________________________________________________________________________________
+
+In a standard setup, you generally have an origin and an upstream remote — the
```
Comment (dylanaraps) [inline vcs/git.txt:67]: `—` -> `-` 

I'd love to use this fancier stuff though mono-space font support outside of ASCII is surprisingly horrible on a lot of mobile devices! Even box drawing characters are unusable for this reason. We could of course ignore this possibility and allow the odd breakage. Will think about it.

**Comment by mcpcpc** [comment]
> This is probably better under `software` as well. The software `index.txt` file can be broken down into "cosmetic" subsections later on.

moved to the appropriate "software" directory.

File: software/git.txt [hunk 4]
```diff
@@ -9,8 +9,8 @@ Commands
 ________________________________________________________________________________
 
 New KISS users may have various levels of exposure to git and other forms of
-version control systems. The following is intended to be "quick" reference sheet
-of git commands.
+version control systems. The following is intended to be a"quick" reference
```
Comment (dylanaraps) [inline software/git.txt:12]: `a"quick"` -> `a "quick"`

**Comment by dylanaraps** [comment]
Thanks! I'll handle the tiny nitpicks :P

**Pull request merged**